lib/repl.js

```diff
@@ -561,6 +561,13 @@ const requireRE = /\brequire\s*\(['"](([\w\.\/-]+\/)?([\w\.\/-]*))/;
 const simpleExpressionRE =
     /(([a-zA-Z_$](?:\w|\$)*)\.)*([a-zA-Z_$](?:\w|\$)*)\.?$/;
 
+function int_filter(item) {
+  return isNaN(parseInt(item));
```

Comment:
Can we use a regex check here, like `/^[A-Za-z_]/.test(item)`, so that the intention is clear?